calcom/cal.com
# fix: removed plain chat from app.cal.com/video

## What does this PR do?



- Fixes #XXXX (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

Make sure to add: 
NEXT_PUBLIC_PLAIN_CHAT_EXCLUDED_PATHS="/video"

## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings
